api/subscriptions/block_reader_test.go

```diff
@@ -24,10 +23,6 @@ import (
 	"github.com/vechain/thor/v2/txpool"
 )
 
-func init() {
-	log15.Root().SetHandler(log15.DiscardHandler())
-}
-
```

Comment:
Removing this doesn't create a butload of messages in the tests does it ?

Reply:
No, because the default logger does not do anything. It has to be initialised to work

Reply:
Perfect 👍